resources/assets/js/home.js

```diff
@@ -393,9 +393,14 @@ window.PoliceScorecardHome = window.PoliceScorecardHome || (function () {
             }, 'admin-0-country-borders');
 
             // Join the sheriff data with the lookup data based on FIPS code
-            function setCounties(e) {
+            function setCounties() {
+                if (!lookupData || !sheriffData || typeof sheriffData.features === 'undefined') {
```

Comment:
renaming 'lookupData' to something more specific could make this code more readable.

I had to trace back to the definition of MB within home.blade.php to discover this data is the MapBox boundaries data. This surprises me since I thought we didn't need to join data for sheriff county views?? I thought we relied on uploading to MapBox for all of the data to be tiled.

Reply:
Ya, this is the code MapBox gave us and I was going to use what they provided since they might be helping us further with map issues.  But fine to rename it.  It is actually a lookup table, so the name makes sense to me.  MapBox did not handle the Sheriff data the same way as the police data, meaning we do not have pre-rendered tiles for Sheriffs, and need to use GeoJSON here.

Reply:
I agree it's a lookup table. The missing information is what it holds, so developers better understand when and how to use it.

Re joining, I must have confused police vs sheriff departments. Thanks for clarifying.

Reply:
Ya, looking over this code that was given to us, that `lookupData` is actually a mess, like, they pass it in as a function param, then use it as a variable within a function inside this other function, and also use it as a constant ... just super confusing.  I will rework this to clean it up.